packages/identitykit/src/libs/react/ui/input/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { useContext } from "react"
+import { isValidURL } from "../../utils"
+import { useForm } from "react-hook-form"
+import { Button } from "../button"
+import { IdentityKitContext } from "../../context"
+
+export const CustomSignerInput = () => {
```

Comment:
- Please put it on the same level as modal.tsx
- Rename file from `input.tsx` to `signer-input.tsx`

Reason for these - it's not a reusable input component it's a specific feature.